elixir-sdk/lib/assignment_logger.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Eppo.AssignmentLogger do
```

Comment:
Unsure whether to use `Eppo.X` or `EppoSDK.X`
```suggestion
defmodule EppoSDK.AssignmentLogger do
```

Curious what other think

Reply:
Defaulting to call it `EppoSdk` for now, seems most consistent but open to other suggestions